packages/graphql-server/src/createGraphQLYoga.ts

```diff
@@ -144,7 +147,8 @@ export const createGraphQLYoga = ({
       useFilterAllowedOperations(allowedOperations || defaultAllowedOperations)
     )
 
-    if (trustedDocuments && !trustedDocuments.disabled) {
+    const useTrustedDocuments = rwjsConfig.graphql?.trustedDocuments
```

Comment:
Makes sense. Thanks for the improvement.

Reply:
I got tired of having to comment stuff in/out every time I toggled the setting in redwood.toml just to try the possibleTypes import logic and the vite plugin 🙂